src/pretix/control/views/organizer.py

```diff
@@ -2310,6 +2311,15 @@ def get_context_data(self, **kwargs):
             o.computed_payment_refund_sum = annotated.get(o.pk)['computed_payment_refund_sum']
             o.icnt = annotated.get(o.pk)['icnt']
             o.sales_channel_obj = scs[o.sales_channel]
+            currency = o.event.currency
+            if currency in spending:
+                total = spending[currency]
+                total += o.total
+                spending[currency] = total
```

Comment:
Könnte man kürzen
```suggestion
                spending[currency] += o.total
```

Reply:
good improvement, though it's completely reworked now :smile: